cmake/SetupJemalloc.cmake

```diff
@@ -16,3 +11,9 @@ file(APPEND
   "${CMAKE_BINARY_DIR}/LibraryVersions.txt"
   "jemalloc Version:  ${JEMALLOC_VERSION}\n"
   )
+
+add_library(Jemalloc::jemalloc INTERFACE IMPORTED)
```

Comment:
Why are you adding the namespace, as opposed to just naming the library `Jemalloc`?

Reply:
The namespacing is what a lot of libraries are using nowadays, so I think it's good to be consistent. Having said that, a lot of our other single-target libs don't do that, so I don't really have a strong argument. I'm fine either way, I'll let you decide.

Reply:
Switched to no namespace as per our slack discussion